specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -522,16 +522,22 @@
           "description": "The status of the email",
           "type": "string",
           "enum": [
-            "Failed",
+            "Bounced",
             "Delivered",
+            "Failed",
             "FilteredSpam",
-            "Quarantined"
+            "Quarantined",
```

Comment:
Consider adding descriptions for each value - https://azure.github.io/autorest/extensions/#x-ms-enum

Reply:
Added descriptions for enum values.